ui/app/AppLayouts/Wallet/controls/ManageTokensDelegate.qml

```diff
@@ -103,7 +102,10 @@ DropArea {
                     root.controller.saveSettings()
                 }
                 onShowHideGroupRequested: function(groupId, flag) {
-                    root.controller.showHideGroup(groupId, flag)
+                    if (isCommunityAsset)
+                        root.controller.showHideGroup(groupId, flag)
+                    else
```

Comment:
What about community collectibles here?

Reply:
Yeah, it's the 1st `if (communityAsset)` branch